asciidoc-release-notes/adoc/source-code.adoc

```diff
@@ -0,0 +1,16 @@
+
+include::entities_all.adoc[]
+include::entities_product.adoc[]
+
+[#sourceCode]
+== Obtaining Source Code
+
+This {suse} product includes materials licensed to {suse} under the GNU
+General Public License (GPL).
+The GPL requires {suse} to provide the source code that corresponds to the
+GPL-licensed material.
+// bsc#1150672
+The source code is available for download at https://www.suse.com/products/server/download/ on Medium{nbsp}2.
```

Comment:
The download URL should probably be configurable from entities-product.adoc, this is not the same across products (but it's always Medium 2).

Reply:
Fixed.